### DISCUSSION TITLE
Commit Drop Down

### DISCUSSION THREAD

**Discussion by rebeccazee** [gitkraken/vscode-gitlens#2073] [Q&A]
Today, I just noticed a drop down on the commit button in the VS Code. I am wondering what it is for. When I click on it does not seem to do anything. This is what the button looks like:
<img width="221" alt="GitLens-commit-button" src="https://user-images.githubusercontent.com/24459985/177421224-a5bf610b-522c-4922-90f4-96fab26f880c.png">.

**Comment by eamodio** [comment]
That is provided by VS Code not GitLens, but it is just a bigger version of the normal commit action -- it will commit your staged changes.
